### PR TITLE
FLUID-6288 Add Dockerfile

### PR DESCRIPTION
**Pull request by gtirloni**
How to test:

```
$ docker build -t musicmania .
$ docker run --rm -ti -p 8000:80 musicmania
$ open http://localhost:8000
```

### CODE REVIEW

**Comment by gtirloni** [comment]
CC @waharnum @jobara

**Comment by BlueSlug** [comment]
Builds and runs fine in my environment.

I suggest incorporating Docker build instructions into the `README.md`

**Comment by gtirloni** [comment]
> I suggest incorporating Docker build instructions into the `README.md`

Thanks, done!

File: README.md [hunk 1]
```diff
@@ -21,6 +21,20 @@ Play game [here](https://build-odeglceahu.now.sh/)
...
+## Running Locally with Docker
+
+You can serve the website from a [Docker](https://docs.docker.com/get-docker) container.
+
+Once you have Docker installed, run the following commands to build a Docker image and start a container:
+
+* Build the image: `docker build -t musicmania .`
+* Run the container: `docker run --name musicmania -p 8000:80 musicmania`
+
+The documentation will be available at [http://localhost:8000](http://localhost:8000)
```
Comment (BlueSlug) [inline README.md]: "documentation" should read "website"

**Review by BlueSlug — APPROVED** [review]
Looks good to me! I can't merge it, unfortunately.

**Pull request merged**